v2/tools/generator/internal/codegen/pipeline/export_generated_code.go

```diff
@@ -8,13 +8,14 @@ package pipeline
 import (
 	"context"
 	"fmt"
-	"golang.org/x/exp/slices"
-	"golang.org/x/sync/errgroup"
 	"os"
 	"path/filepath"
 	"sync"
 	"time"
 
+	"golang.org/x/exp/slices"
+	"golang.org/x/sync/errgroup"
```

Comment:
minor: I think this is correct since these aren't stdlib imports, but they can be grouped with the group below. (there should just be 3 groups total AFAIK: stdlib, external, internal)